PhysicsTools/NanoAOD/python/jetsAK8_cff.py

```diff
@@ -317,10 +317,50 @@ def nanoAOD_addDeepInfoAK8(process, addDeepBTag, addDeepBoostedJet, addDeepDoubl
 fatJetTable.variables.pt.precision=10
 subJetTable.variables.pt.precision=10
 
+##############################################################
+# AK8 constituents
+###############################################################
+finalJetsAK8PFConstituents = cms.EDProducer("PatJetConstituentPtrSelector",
+    src = cms.InputTag("finalJetsAK8"),
+    cut = cms.string("abs(eta) <= 2.5")
```

Comment:
Would it be better to use the following to ensure consistency?
```suggestion
    src = fatJetTable.src,
    cut = fatJetTable.cut
```

Reply:
For `src`,  you are right. I will correct it but for `cut`, we explicitly set `"abs(eta) <= 2.5"` to only store the PF candidates for AK8 jets within that eta range.

Reply:
Why explicitly requiring `abs(eta) <= 2.5`? IIRC in MiniAOD it's already `rapidity<2.4`, so it's largely similar but there could be some edge cases?

Reply:
There is a selection on the jets which we retrieve the their constituents to run soft-drop grooming: 

https://github.com/cms-sw/cmssw/blob/6f9d2fd122dd1a13861bc581be8d87034489f8d9/PhysicsTools/PatAlgos/python/slimming/applySubstructure_cff.py#L36-L37

and then a `pt > 100` selection to store slimmedJetsAK8:
 
https://github.com/cms-sw/cmssw/blob/6f9d2fd122dd1a13861bc581be8d87034489f8d9/PhysicsTools/PatAlgos/python/slimming/applySubstructure_cff.py#L131

so we can still have AK8 jets in the forward region as I cannot find any explicit eta cut for AK8 puppi jets in CMSSW.